docs/querying-data.rst

```diff
@@ -0,0 +1,153 @@
+----------------------------------------
+
+Use the collection ``id`` returned by the previous query to restrict your analysis to a single collection.
+
+The following query returns the full JSON data for the first 3 compiled releases in collection 584:
+
+.. code-block:: sql
+
+  select
+      data
+  from
+      data --raw OCDS JSON data is stored as jsonb blobs in the `data` column of the `data` table
+  join
+      compiled_release
+  on
+    data.id = compiled_release.data_id -- join to the `compiled_release` table to filter data from a specific collection
+  where
+      collection_id = 584
+  limit 3;
+
+To get data from a different collection, change the ``collection_id`` parameter.
+
+To get data from a collection containing releases or records, join to the ``release`` or ``record`` tables rather than the ``compiled_release`` table.
+
+.. tip:: Rendering JSON using Redash
+
+  If you are using Redash, you can render the results of a query as pretty printed and collapsible JSON by clicking the '+ New Visualization' button, setting the visualization type to 'table' and setting the data column to display as JSON.
+
+Calculate the total value of completed tenders in a collection
+--------------------------------------------------------------
+
+In OCDS, the tender value is stored in the ``tender.value`` object which consists of a numeric ``.amount`` property with an associated ``.currency``. The tender status is stored in the ``tender.status`` field.
+
+To access the properties of a JSON object use the PostgreSQL ``->`` operator. The ``->`` operator takes a jsonb object and the name of a key as inputs and returns the value of the key as a jsonb value. The ``->>`` operator returns the value as a string.
+
+
+The following query calculates the total value of completed tenders in collection 584:
+
+.. code-block:: sql
+
+  select
+      sum((data -> 'tender' -> 'value' -> 'amount')::numeric) as tender_value,
+      data -> 'tender' -> 'value' ->> 'currency' as currency
+  from
+      data
+  join
+      compiled_release
+  on
+      data.id = compiled_release.data_id
+  where
+      collection_id = 584
+  and
+      data -> 'tender' ->> 'status' = 'complete'
+  group by
+      currency;
+
+.. tip:: Filtering on status fields
+
+  The ``tender``, ``award`` and ``contract`` objects in OCDS all have a ``.status`` property.
+
+  Consider which statuses you want to include or exclude from your analysis, for example you might wish to exclude pending and cancelled contracts when calculating the total value of contracts for each buyer.
+
+  The `OCDS codelist documentation <https://standard.open-contracting.org/latest/en/schema/codelists/#>`__ describes the meaning of the statuses for each object.
+
+Calculate the top 10 buyers by award value
+------------------------------------------
+
+Details of the buyer for a contracting process in OCDS are stored in the ``parties`` section and referenced from the ``buyer`` object.
```

Comment:
Are we presuming knowledge of `OrganizationReference`? It might be worth just adding "via the Organization Reference \<link to standard docs\> object". I feel like for those new to analysing OCDS this might make a big difference. Alternatively if we have dedicated or separate guidance on parties in OCDS that'd be good.

Reply:
Good point, I've linked it to the OCDS docs and similar in a few other places in https://github.com/open-contracting/kingfisher-process/pull/220/commits/0a98606434543a9c5395f3aa673c6a4237bd4ff6